data.js

```diff
@@ -0,0 +1,74 @@
+const projects = [
```

Comment:
[OPTIONAL] You can add a paragraph or detailed description because on the home it is just a short description and on the popup, it is more detailed.